CHANGELOG.md

```diff
@@ -12,6 +12,19 @@ We've deprecated the `govuk-header__link--service-name` class, and will remove i
 
 This change was introduced in [pull request #2617: Do not make the service name in the header a link if no `serviceUrl` is provided](https://github.com/alphagov/govuk-frontend/pull/2617).
 
+#### File extensions added for JavaScript ES Module imports
+In 4.1.0, we started providing our component JavaScript in ES module format. This JavaScript used import statements without [mandatory file extensions](https://nodejs.org/api/esm.html#mandatory-file-extensions), as required by Node.js.
+
+We have now updated our component JavaScript to include these missing file extensions. If you previously received an error similar to the below, this fix should resolve that issue:
+
+```
+Cannot find module '../node_modules/govuk-frontend/govuk-esm/common' imported from ../node_modules/govuk-frontend/govuk-esm/all.mjs
+```
+
+If you are already successfully importing our JavaScript as ES modules, you should not need to make any changes, but there may be config you can remove. For example, removing `fullySpecified: false` from your Webpack config file.
```

Comment:
My only thought is that this is under "Recommended changes", and we get right up to this line before we suggest an actual change.

Are we able to expand on an entry in "Fixes"? Or is that not really something we do? We could possibly include this action here, and the rest of the info under fixes.